src/app/header/header.component.ts

```diff
@@ -11,7 +11,6 @@ import { Broadcaster } from './../shared/broadcaster.service';
   selector: 'alm-app-header',
   templateUrl: './header.component.html',
   styleUrls: ['./header.component.scss'],
-  changeDetection: ChangeDetectionStrategy.OnPush
```

Comment:
@joshuawilson any particular reason that you used change detection strategy on push?